dbms/src/Storages/DeltaMerge/Remote/ObjectId.h

```diff
@@ -51,7 +52,7 @@ struct fmt::formatter<DB::DM::Remote::DMFileOID>
     template <typename FormatContext>
     auto format(const DB::DM::Remote::DMFileOID & value, FormatContext & ctx) const -> decltype(ctx.out())
     {
-        return format_to(ctx.out(), "{}_{}_{}", value.store_id, value.table_id, value.file_id);
+        return format_to(ctx.out(), "{}_{}_{}_{}", value.store_id, value.keyspace_id, value.table_id, value.file_id);
```

Comment:
should it ignore the keyspace_id when NullspaceID?
if (value.keyspace_id == NullspaceID)
    return format_to(ctx.out(), "{}_{}_{}", value.store_id, value.table_id, value.file_id);
else
    return format_to(ctx.out(), "{}_{}_{}_{}", value.store_id, value.keyspace_id, value.table_id, value.file_id);

Reply:
Yes, this is better. Fixed.